CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py

```diff
@@ -74,7 +78,9 @@ def extract_fst_lemmas(
 
             if xml_lemma in xml_lemma_to_pos_lc:
                 for pos, lc in xml_lemma_to_pos_lc[xml_lemma]:
-                    xml_lemma_pos_lc_to_analysis[(xml_lemma, pos, lc)] = ConcatAnalysis("")
+                    xml_lemma_pos_lc_to_analysis[(xml_lemma, pos, lc)] = ConcatAnalysis(
+                        ""
```

Comment:
I think Black is trying to tell us something... 😂

Reply:
black is a paranoid :b: